Dockerfiles/midstream/entrypoint.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash -e
+main() {
+    local username="${SSH_AUTH_USER:-default}"
+    local home
+    home=$(mk_home "$username")
+    setup_passwd "$username" "$home"
+    exec sleep inf
+}
+mk_home() {
+    local username="${1:?}"
+    [[ -d /workDir ]] && [[ -w /workDir ]] && \
+        mkdir -p "/workDir/home/$username" && \
+        echo "/workDir/home/$username" && \
```

Comment:
I would rather create it in `/tmp` than in `/workDir`

Reply:
How would it effect the run for changing to /tmp/ ?
I understand it might be the permissions. 
How would this help ?

Reply:
updating meanwhile in subsequent commits

Reply:
`workDir` is a non-standard directly and we should not rely on it.